nitro-protocol/src/signatures.ts

```diff
@@ -59,6 +62,91 @@ export async function sign(wallet: Wallet, msgHash: string | Uint8Array): Promis
   return utils.splitSignature(await wallet.signMessage(utils.arrayify(msgHash)));
 }
 
+// Towards #761 https://github.com/statechannels/go-nitro/issues/761
+//
+export type ShortenedVariablePart =
+  | {
+      signerIndices: number[];
+      appData?: Bytes;
+      outcome?: Outcome;
+      isFinal?: boolean;
+    }
+  | number[];
+
+// Map<turnNum => ShortenedVariablePart> | Map<turnNum => signerIndices>
+export type TurnNumToShortenedVariablePart = Map<number, ShortenedVariablePart>;
```

Comment:
Here naming becomes even worse... If to use the current `ShortenedVariablePart` name, it can be `ShortenedVariablePartShortHand` which is too confusing.
Any suggestions are hugely appreciated 😄

Reply:
I think what you have is OK, actually (can't think of anything better just now!)